show-land/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WeProDev BeTheme</title>
+
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+
+<div class="container">
+<header class="header">
+
+<div class="logo">logo</div>
+
+<nav class="">nav</nav>
+
+</header>
+<section class="hero">hero</section>
+
+<main>
+
+    <section class="super-fast-section">super fast</section>
+    <section class="Home-page-section">Home page </section>
+    <section class="page-builder-section">page builder section</section>
+    <section class="benefits-section">Benefits section </section>
+    <section class="features-section ">features section </section>
+    <section class="powerful-features-section">Powerful features section</section>
+    <section class="help-users-section">Help users section</section>
+    <section class="trusted-section ">trusted section </section>
+    <section class="footer-section">footer section</section>
+</div>
+
+</main>
+    
+</div>
+</div>
+
+    </body>
+    </html>
```

Comment:
Please use "prettier" to format the code as mentioned in code convention's page :)